README.md

```diff
@@ -31,7 +31,8 @@ To get the Harvester server up and running the following minimum hardware is req
 |:---|:---|
 | CPU | 4 cores minimum, 16 cores or above preferred |
 | Memory | 8 GB minimum, 32 GB or above preferred |
-| Disk |  120 GB minimum, 500 GB or above preferred |
+| Disk Capacity |  120 GB minimum, 500 GB or above preferred |
+| Disk Performance |  Management nodes (first 3 nodes) must be [fast enough for Etcd](https://www.ibm.com/cloud/blog/using-fio-to-tell-whether-your-storage-is-fast-enough-for-etcd), SSD recommended |
```

Comment:
The disk I/O performance of the management nodes(can be checked on the UI's host detail page) should meet the criteria according to [this article](https://www.ibm.com/cloud/blog/using-fio-to-tell-whether-your-storage-is-fast-enough-for-etcd).

Reply:
Due to the node role promote feature, users should make sure that the disks in the first three nodes all meet the performance requirements.